File: SDKs/Aspose.Words_Cloud_SDK_for_.NET/src/Com/Aspose/Words/Model/TextItems.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Words.Model {
  public class TextItems {
    public List<TextItem> List { get; set; }

    public Link link { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class TextItems {\n");
      sb.Append("  List: ").Append(List).Append("\n");
      sb.Append("  link: ").Append(link).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: SDKs/Aspose.Words-Cloud-SDK-for-Java/src/main/java/com/aspose/words/model/TextItems.java
package com.aspose.words.model;

import java.util.*;
import com.aspose.words.model.Link;
public class TextItems {
  private List<TextItem> List = new ArrayList<TextItem>();
  private Link link = null;
  /**
	 * getList
	 * Gets List<TextItem>
	 * @return List
	 */
  public List<TextItem> getList() {
    return List;
  }

	/**
	 * setList
	 * Sets List<TextItem>
	 * @param List List<TextItem>
	 */
  public void setList(List<TextItem> List) {
    this.List = List;
  }

  /**
	 * getLink
	 * Gets Link
	 * @return link
	 */
  public Link getLink() {
    return link;
  }

	/**
	 * setLink
	 * Sets Link
	 * @param link Link
	 */
  public void setLink(Link link) {
    this.link = link;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class TextItems {\n");
    sb.append("  List: ").append(List).append("\n");
    sb.append("  link: ").append(link).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

